rosbag2_bag_v2_plugins/src/rosbag2_bag_v2_plugins/storage/rosbag_v2_storage.cpp

```diff
@@ -104,13 +104,23 @@ std::vector<rosbag2_storage::TopicMetadata> RosbagV2Storage::get_all_topics_and_
   return topics_with_type;
 }
 
+std::string RosbagV2Storage::get_storage_identifier() const
+{
+  return "rosbag_v2";
```

Comment:
Probably should be defined as a constant.

Reply:
`static constexpr const char * const STORAGE_IDENTIFIER = "rosbag_v2";` at the top of this file.

Reply:
It would be nice to get this in before the merge, so those who are calling get_storage_identifier don't have to check against a manually constructed string :-)

Reply:
I actually did:
```c++
namespace
{
const char * IDENTIFIER = "rosbag_v2";
}
```

I'll change it to  ```constexpr const char * const```